microsoft_auth/client.py

```diff
@@ -151,11 +151,11 @@ def get_claims(self, allow_refresh=True):
             claims = jwt.decode(
                 token,
                 public_key,
-                algoithm="RS256",
+                algorithms="RS256",
```

Comment:
Should the value of algorithms be a list of strings instead of a string? https://pyjwt.readthedocs.io/en/1.7.1/#example-usage

Reply:
Looks like it should be. I am actually surprised this ever worked. Haha.

Reply:
Fix a typo, create a new typo... I'll take a proper look in the next day or so and update the PR.